modules/javafx.graphics/src/main/native-glass/mac/GlassWindow.m

```diff
@@ -843,7 +843,9 @@ static jlong _createWindowCommonDo(JNIEnv *env, jobject jWindow, jlong jOwnerPtr
         }
         else
         {
-            [window->nsWindow performSelectorOnMainThread:@selector(setContentView:) withObject:nil waitUntilDone:YES];
+            // If the contentView is set to nil within performKeyEquivalent: the OS will crash.
+            NSView* dummy = [[NSView alloc] initWithFrame: NSMakeRect(0, 0, 10, 10)];
+            [window->nsWindow performSelectorOnMainThread:@selector(setContentView:) withObject:dummy waitUntilDone:YES];
```

Comment:
I'm not sure is needed or worth it, but maybe the `dummy` view should be released?

Reply:
That's a good question.

Reply:
Yes, it should be released. I will update the PR.